Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Runs on the default latest node LTS image
+# Build stage
+FROM node:lts AS build
+
+## Create the solid community server directory
+RUN mkdir /community-server
+
+## Set current working directory
+WORKDIR /community-server
+
+## Copy the dockerfile's context's community server files
+COPY . .
```

Comment:
I think you still want to include the two stage package.json copy trick I mentioned in the other PR, as it will improve the build time of this image significantly. Otherwise the dependencies will be rebuilt every time there is a code change, even if there is no change in the dependencies. I got the suggestion from the Nodejs guide on Dockerizing node apps: https://nodejs.org/en/docs/guides/nodejs-docker-webapp/

Reply:
Good point!